app/models/IUser.ts

```diff
@@ -1,12 +1,5 @@
-export interface IApiUser {
-  token: string;
-  id: string;
-  name: string;
-  surname: string;
-  email: string;
-  phone: string;
-  image?: string;
-  email_verified_at: string;
-  create_at: string;
-  updated_at: string;
-}
+import { FirebaseAuthTypes } from '@react-native-firebase/auth';
+
+import { Nullable } from '../utils';
+
+export type IUser = Nullable<FirebaseAuthTypes.User>;
```

Comment:
User's model is `FirebaseAuthTypes.User | null`?
Model shouldn't have `null` types